controller/src/main/java/io/pravega/controller/store/kvtable/PravegaTablesKVTable.java

```diff
@@ -191,11 +191,14 @@ private String getMetadataTableName(String id) {
 
     @Override
     CompletableFuture<Void> createKVTableMetadata() {
-        return getId().thenCompose(id -> {
-            String metadataTable = getMetadataTableName(id);
-            return CompletableFuture.allOf(storeHelper.createTable(metadataTable))
-                    .thenAccept(v -> log.debug("kvtable {}/{} metadata table {} created", getScopeName(), getName(), metadataTable));
-        });
+        return getId().thenCompose(id -> CompletableFuture.allOf(storeHelper.createTable(getMetadataTableName(id)))
```

Comment:
there only seems to be one future.. why CompletableFuture.allOf?

Reply:
fixed